<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StraightRebar</class>
 <widget class="QDialog" name="StraightRebar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="heading">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fill all the values</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QPushButton" name="submit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="rebarAlongLabel">
       <property name="text">
        <string>Rebar Along</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="rebarAlong"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="frontCoverLabel">
       <property name="text">
        <string>Front Cover</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="frontCover"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="sideCoverLabel">
       <property name="text">
        <string>Side Cover</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="sideCover"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="bottomCoverLabel">
       <property name="text">
        <string>Bottom Cover</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="bottomCover"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="bentLengthLabel">
       <property name="text">
        <string>Bent Length</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="bentLength"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="diameterLabel">
       <property name="text">
        <string>Diameter</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="diameter"/>
     </item>
     <item row="7" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="amount_radio">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="spacing_radio">
         <property name="text">
          <string>Spacing</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="amountLabel">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="amount">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="spacingLabel">
       <property name="text">
        <string>Spacing</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="spacing">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="roundingLabel">
       <property name="text">
        <string>Rounding</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="rounding"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
